lib/main.dart

```diff
@@ -1,125 +1,141 @@
+                        child: TitleWidget(
+                          movie: movie,
+                        ),
+                      ),
+                    ],
+                  ),
+                  Padding(
+                    padding: const EdgeInsets.all(UiConstants.padding16),
+                    child: Row(
+                      mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                      children: [
+                        MiddleText(
+                            text:
+                                '${UiConstants.textConstant} ${movie.releaseDate}'),
+                        MiddleText(text: movie.voteAverage),
+                      ],
+                    ),
+                  ),
+                  Row(
+                      mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                      children: movie.genres
+                          .map(
+                            (genre) => Container(
+                              decoration: const BoxDecoration(
+                                color: Colors.black12,
+                                borderRadius: BorderRadius.all(
+                                  Radius.circular(UiConstants.borderRadius),
+                                ),
+                              ),
+                              child: Padding(
+                                padding:
+                                    const EdgeInsets.all(UiConstants.padding8),
+                                child: Text(
+                                  genre,
+                                  style: CustomTextStyle.customTextStyleWhite,
+                                ),
+                              ),
+                            ),
+                          )
+                          .toList()),
+                  const SizedBox(
+                    height: UiConstants.sizedBoxHeight,
+                  ),
+                  Stack(
+                    children: [
+                      Image.network(movie.poster),
+                      Positioned(
+                        top: UiConstants.secondPositionedTop,
+                        right: UiConstants.secondPositionedRight,
+                        child: Column(
+                          children: [
+                            const Icon(
+                              Icons.favorite_outlined,
+                              size: UiConstants.iconSize,
+                              color: Colors.red,
+                            ),
+                            FloatingActionButton(
+                              onPressed: () {},
+                              child: const Icon(Icons.add),
+                            ),
+                          ],
+                        ),
+                      ),
+                      Padding(
+                        padding: const EdgeInsets.symmetric(
+                          vertical: UiConstants.padding150,
+                          horizontal: UiConstants.padding16,
```

Comment:
We don't recommend to use number when we define const

Reply:
done